test/files/pos/t11681.scala

```diff
@@ -0,0 +1,8 @@
+//
+// scalac: -Wunused:privates -Werror
```

Comment:
Is this an alternative method to pass flags to the compiler in the `partests`? Was it recently added?

Reply:
No, that was a long time ago in internet time. I noticed yesterday (since Seth commented) that my upgrade to jdk10 PR included many such cleanups.